WordPress/Classes/System/RootViewPresenter+EditorNavigation.swift

```diff
@@ -16,18 +16,14 @@ extension RootViewPresenter {
     func showPostTab(completion afterDismiss: (() -> Void)?) {
         let context = ContextManager.shared.mainContext
         // Ignore taps on the post tab and instead show the modal.
-        if Blog.count(in: context) == 0 {
-            mySitesCoordinator.showAddNewSite()
```

Comment:
I decided to remove it because I don't think we need to support this complication. I'm not sure I follow what the scenario is. If you have no blogs, it means that either you have no sites – they where are you opening the "posts" link from, or that for some reason sites weren't synced yet.

Reply:
From a quick code search, this function can be called from a few places like Stats and Reader. Calling from Reader is probably the only scenario where the account doesn't have any site.

Reply:
I'll approve this PR, as I don't think it's super important to keep this line. But feel free to add it back if you find it necessary.